packages/directives/repeat-click/index.ts

```diff
@@ -1,27 +1,26 @@
-// @ts-nocheck
-import { on, once } from '@element-plus/utils'
-
 import type { DirectiveBinding, ObjectDirective } from 'vue'
 
-export default {
+const RepeatClick: ObjectDirective = {
   beforeMount(el: HTMLElement, binding: DirectiveBinding) {
-    let interval = null
+    let interval: ReturnType<typeof setInterval> | null = null
     let startTime: number
     const handler = () => binding.value && binding.value()
     const clear = () => {
       if (Date.now() - startTime < 100) {
         handler()
       }
-      clearInterval(interval)
+      clearInterval(interval!)
       interval = null
     }
 
-    on(el, 'mousedown', (e: MouseEvent) => {
-      if ((e as any).button !== 0) return
+    el.addEventListener('mousedown', (e: MouseEvent) => {
+      if (e.button !== 0) return
       startTime = Date.now()
-      once(document as any, 'mouseup', clear)
-      clearInterval(interval)
+      document.addEventListener('mouseup', clear)
```

Comment:
Are the changes here conflicting with the previous ones?

Reply:
no

Reply:
IMO, it is inconsistent with the previous behavior. because the EventListener is automatically removed before this change, but after this change it was not removed. This change will cause every mouseup event on the document to trigger the clear function.

Reply:
Sorry for forgetting to set the `once` option.